docs/lang/articles/advanced/dataclass.md

```diff
@@ -6,13 +6,13 @@ sidebar_position: 4
 
 Taichi provides custom [struct types](../type_system/type.md#compound-types) for developers to assemble pieces of data together. However, it would be more convenient to have:
 
-- A Python representation of the struct type which is more object oriented.
-- Functions associated with a struct type (C++-style structs).
+- A Python representation of the struct type which is more Object-Oriented.
+- Functions associated with a struct type.
 
 
 To achieve the ends, Taichi enabled the `@ti.dataclass` decorator on a Python class. This is inspired by Python's [dataclass](https://docs.python.org/3/library/dataclasses.html) feature, which uses class fields with annotations to create data types.
 
-## Create a struct from a Python class
+## Create a Struct from a Python Class
```

Comment:
Let's use sentence case for subheadings :)

Reply:
I think headings should have the relevant words capitalized. We will discuss in upcoming meeting